assets/sass/protocol/base/elements/_forms.scss

```diff
@@ -42,79 +49,73 @@ textarea {
 }
 
 button,
-[type="button"],
-[type="reset"],
-[type="submit"] {
-    -webkit-appearance: button;
+[type='button'],
+[type='reset'],
+[type='submit'] {
+    appearance: button;
 }
 
 // Remove the default vertical scrollbar in IE 10+.
 textarea {
     overflow: auto;
+    height: auto;
 }
 
 // 1. Correct the odd appearance in Chrome and Safari.
 // 2. Correct the outline style in Safari.
-[type="search"] {
-    -webkit-appearance: textfield; // 1
+[type='search'] {
+    appearance: textfield; // 1
     outline-offset: -2px; // 2
 }
 
 // Remove the inner padding in Chrome and Safari on macOS.
-[type="search"]::-webkit-search-decoration {
-    -webkit-appearance: none;
+[type='search']::-webkit-search-decoration {
+    appearance: none;
```

Comment:
Some of these rules are specifically for older versions of Safari, so we might still need the prefixed property when it's also using a webkit-only selector. I could be wrong, we should probably research these a bit.

Reply:
for this one lets add a comment on the line above to disable the particular rule since its a bit of a one-off:

```css

 /* stylelint-disable-next-line value-no-vendor-prefix */
 -webkit-appearance: none;

```